lib/components/animation_component.dart

```diff
@@ -1,22 +1,22 @@
 import 'dart:ui';
 
 import 'component.dart';
-import 'package:flame/animation.dart';
+import 'package:flame/sprite_animation.dart';
```

Comment:
Maybe we should change this import to be relative, so we keep it on the standard of the other imports?

Reply:
As I have seen on other components, import of files of the first level of teh package are made this way.

Reply:
Not really, check Sprite.dart for example, all imports a relative, the imports that are absolute from the package are some old ones, anyway this is just a suggestion, this isn't really related to your PR.

Reply:
wilco